README.md

```diff
@@ -110,7 +110,8 @@ _this is the recommended way to run remark42_
 | store.type              | STORE_TYPE              | `bolt`                   | type of storage, `bolt` or `rpc`                |
 | store.bolt.path         | STORE_BOLT_PATH         | `./var`                  | path to data directory                          |
 | store.bolt.timeout      | STORE_BOLT_TIMEOUT      | `30s`                    | boltdb access timeout                           |
-| admin.shared.id         | ADMIN_SHARED_ID         |                          | admin names (list of user ids), _multi_         |
+| admin.shared.id         | ADMIN_SHARED_ID         |                          | admin ids (list of user ids), _multi_           |
+| admin.shared.name       | ADMIN_SHARED_NAME       |                          | admin names, _multi_                            |
```

Comment:
LGTM except for this line.

It's even `admin.shared.name`, it's "list of names restricted to use by anonymous users and maybe by email users in the future if we'll change our mind". We should add a proper description to that new option and (maybe) move it somewhere else, as it has nothing to do with admins other than it's recommended use for that functionality.

Reply:
yeah, bothered me as well. The thoughts behind that choice were - such names may be used not just to limit auth but for something else, like get the list of admin names. However I tend to agree with you because there is no any relation between names and ids, so if we even add the list of names it won't be associated with ids.

Reverting this part and moving it to restricted-names param